### ISSUE TITLE
allow label "milestone" for wrapper --gradle-version

### ISSUE THREAD

**Issue by reinsch82** [gradle/gradle#23369]
this is a follow up of
- #13504
and
-  #23179

This should work very similar to the "release-candidate" label. But since there is no explicit endpoint in `https://services.gradle.org/versions` that provides the latest milestone some calculation has to be done to find the latest milestone out of all available versions

**Comment by jbartok** [comment]
Thank you for your interest in Gradle!

This feature request is in the backlog of the relevant team but this area of Gradle is currently not a focus one so it might take a while before it gets implemented.

**Comment by sschuberth** [comment]
> Thank you for your interest in Gradle!

@jbartok, are you talking to @reinsch82 who seems to be a Gradle employee himself? That's funny 😁

**Comment by jbartok** [comment]
Yes he is, we're trying to be consistent in our triage process. But it is.
